src/services/roverPhotos.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+const { format, subDays } = require('date-fns');
+require('dotenv').config();
+
+const { API_BASE_URL, MARS_PHOTO_ENDPOINT, MANIFESTS_ENDPOINT } = process.env;
+const { API_KEY } = process.env;
```

Comment:
could you create config file with all env variables?
in the future you will be able to validate variables